### PR TITLE
Drupal 10.1

### PR DESCRIPTION
**Pull request by back-2-95**
- Update Drupal core to 10.1

Note: Drupal 10.1 has now latest Symfony 6.3 packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ the modern app server for PHP.
 
 ## Getting Started
 
-```
+```shell
```
Comment (dunglas) [inline README.md]:
```suggestion
```console
```

(Nitpicking)

File: docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Access via "https://localhost"
 #
 # During initial Drupal setup,
-# Database type: PostgreSQL
+# Database type: PostgresSQL
```
Comment (dunglas) [inline docker-compose.yml]:
```suggestion
# Database type: PostgreSQL
```

To revert, the original name is good.

**Review by dunglas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (back-2-95) [reply README.md]: I did not know this, is this documented somewhere (on Github)?

Comment (dunglas) [reply README.md]: I don't know but this allows to have the text retuned by the command on another color.

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thank you!

**Pull request merged**